openvdb_cmd/vdb_tool/README.md

```diff
@@ -0,0 +1,327 @@
+# vdb_tool
+
+This command-line tool, dubbed vdb_tool, can combine any number of the of high-level tools available in openvdb/tools. For instance, it can convert a sequence of polygon meshes and particles to level sets, and perform a large number of operations on these level set surfaces. It can also generate adaptive polygon meshes from level sets, ray-trace images and export particles, meshes or VDBs to disk or even stream VDBs to STDOUT so other tools can render them (using pipelining). We denote the operations **actions**, and their arguments **options**. This command-line tool also supports a string-evaluation language that can be used to define procedural expressions for options of the actions. Currently the following list of actions are supported:
```

Comment:
We probably want to emphasize early on that it can be used to generate and read configuration files! This seems to be a very useful workflow where a user can share a mesh file, followed by a vdb_tool configuration file to turn it into a desired vdb/nanovdb.